File: flutter-examination/lib/model/place.dart
class Place {
  final int id;
  final String name;
  final String image;
  final double rate;
  final bool favorite;

  Place({
    required this.id, required this.name, required this.image, required this.rate, required this.favorite
  });

  factory Place.fromJson(Map<String, dynamic> json) {
    return Place(
      id: json['id'],
      name: json['name'],
      image: json['image'],
      rate: json['rate'],
      favorite: json['favorite']
    );
  }
}

File: demo/src/main/java/com/example/demo/Place.java
package com.example.demo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity(name = "place")
public class Place {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private Long id;
    private String name;
    private String image;
    private double rate;
    private boolean favorite;
}
